### DISCUSSION TITLE
When server rendering, you must wrap your application in an <SSRProvider> to ensure consistent ids are generated between the client and server.

### DISCUSSION THREAD

**Discussion by armatrix** [heroui-inc/heroui#1036] [Help]
here's my _app.js:
` <WagmiConfig client={wagmiClient}>
      <RainbowKitProvider chains={chains} theme={darkTheme()}>
        <ConfigProvider>
          <NextUIProvider attribute="class" {...theme}>
            <Layout>
              <Component {...pageProps} />
            </Layout>
          </NextUIProvider>
        </ConfigProvider>
      </RainbowKitProvider>
      <Toaster />
    </WagmiConfig>`

but it keep showing "When server rendering, you must wrap your application in an <SSRProvider> to ensure consistent ids are generated between the client and server."

**Comment by jrgarciadev** [comment]
Hey @armatrix  sorry for the delay please upgrade to [V2](https://nextui.org/docs/guide/upgrade-to-v2)
